cost/aws/elb/clb_unused/AWS_Delete_Unused_CLB.pt

```diff
@@ -0,0 +1,243 @@
+name "AWS Delete Unused Elastic Load Balancers (CLB)"
+rs_pt_ver 20180301
+type "policy"
+short_description "Report and remediate any  ELB (CLB) that are not currently in use. \n See the [README](https://github.com/rightscale/policy_templates/tree/master/cost/aws/elb/clb_unused) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
+long_description "Version: 1.0"
+category "Cost"
+severity "low"
+
```

Comment:
@srpomeroy -Since I am not using any CM resources in this policy, I am not sure what all resources and actions to be included under the permission statement, Can you help me in understanding more on the permissions to be considered in this policy?

Reply:
You are using credentials, which are CM resources.
At the least the selected code in the example should be added to the policy.

Reply:
Added the selected code to the policy,  Thanks.